src/editors/containers/VideoUploadEditor/index.jsx

```diff
@@ -11,77 +12,69 @@ import * as hooks from './hooks';
 import messages from './messages';
 import * as editorHooks from '../EditorContainer/hooks';
 
-export const VideoUploader = ({ onUpload, errorMessage }) => {
-  const { textInputValue, setTextInputValue } = hooks.uploader();
-  const onURLUpload = hooks.onVideoUpload();
+const URLUploader = () => (
+  <div className="d-flex flex-column flex-wrap">
+    <div className="justify-content-center align-self-center bg-light rounded-circle p-4">
+      <Icon src={FileUpload} className="text-muted" />
+    </div>
+    <div className="d-flex align-self-center justify-content-center flex-wrap flex-column pt-5">
+      <span style={{ fontSize: '20px' }}><FormattedMessage {...messages.dropVideoFileHere} /></span>
+      <span className="align-self-center" style={{ fontSize: '12px' }}><FormattedMessage {...messages.info} /></span>
+    </div>
+    <div className="align-self-center justify-content-center mx-2 text-dark">OR</div>
+  </div>
+);
 
-  const { getRootProps, getInputProps, isDragActive } = useDropzone({
-    accept: 'video/*',
-    multiple: false,
-    onDrop: (acceptedFiles) => {
-      if (acceptedFiles.length > 0) {
-        const uploadfile = acceptedFiles[0];
-        onUpload(uploadfile);
-      }
-    },
-  });
+export const VideoUploader = ({ onUpload }) => {
+  const { textInputValue, settextInputValue } = hooks.uploader();
+  const onURLUpload = hooks.onVideoUpload();
+  const URLUploaderComponent = URLUploader();
 
-  const handleInputChange = (event) => {
-    setTextInputValue(event.target.value);
+  const handleInputChange = (value) => {
```

Comment:
This function isn't doing anything, just set onChange to setTextInputChange directly.